src/components/create-user.html

```diff
@@ -115,18 +121,83 @@ <h1 class="div-center">Create User</h1>
                 id: ''
               }
             }
+          },
+          hasError: {
+            type: Boolean,
+            value: false
+          }, 
+          firstNameError: {
+            type: String,
+            value: ''
+          }, 
+          lastNameError: {
+            type: String,
+            value: ''
+          }, 
+          phoneError : {
+            type: String,
+            value: ''
+          }, 
+          emailError: {
+            type: String,
+            value: ''
           }
         }
       }
-        
+
+      getUsers() {
+        this.dispatchEvent(new CustomEvent('updateUserList', { bubbles: true, composed: true }));
+      }
+
       createUser() {
-        this.storage.createUser(this.user);
+        this.validateInput();
+        if(!this.hasError) {
+          this.storage.createUser(this.user);
+        }
+        this.hasError = false;
+        this.dispatchEvent(new CustomEvent('updateUserList', { bubbles: true, composed: true }));
       }
 
-      updateUserList() {
-        this.createUser();
+      deleteAllUsers() {
+        this.storage.deleteAllUsers();       
         this.dispatchEvent(new CustomEvent('updateUserList', { bubbles: true, composed: true }));
       }
+
+      validateInput() {
```

Comment:
this could probably be split out into multiple functions, and probably should be.

Reply:
Yeah I agree for sure. Will do.